.github/workflows/go.yml

```diff
@@ -35,6 +35,9 @@ jobs:
     - name: Get Dependencies (github.com/golang.org/x/tools/imports)
       run: go get -v ./...
 
+    - name: Get go-cmp
+      run: go get github.com/google/go-cmp/cmp
```

Comment:
Why do we need to manually fetch this package? If a user forgets to do this will gotests not work for them? IMO `go get -v ./...` should cover installing this dependency.

Reply:
Without this explicit call the tests will fail in GitHub Actions for Go 1.9 - 1.12:

```
Run go test -v ./...
  go test -v ./...
  shell: /usr/bin/bash -e {0}
  env:
    GOPATH: /home/runner/work/gotests/gotests
    GO111MODULE: auto
    GOROOT: /opt/hostedtoolcache/go/1.12.17/x64
# github.com/cweill/gotests
FAIL	github.com/cweill/gotests [setup failed]
gotests_test.go:16:2: cannot find package "github.com/google/go-cmp/cmp" in any of:
	/opt/hostedtoolcache/go/1.12.17/x64/src/github.com/google/go-cmp/cmp (from $GOROOT)
	/home/runner/work/gotests/gotests/src/github.com/google/go-cmp/cmp (from $GOPATH)
FAIL	github.com/cweill/gotests/gotests [setup failed]
```
 
Other solutions to make this work may exist. The `GO111MODULE` env var could be set to `on`, but that'll likely only work for Go 1.11 and up. To me this seemed the easiest solution to make it work across versions without having to make the CI configuration more complex.

I tried the `go-cmp` version of `gotests` locally on a sample project. It resulted in a `_test.go` file with `github.com/google/go-cmp/cmp` in the import section. Running the tests for the first time results in `go-cmp` being inserted into go.mod. The go.mod of gotests itself does contain `go-cmp`, so that should work correctly too.

Reply:
My concern is for who don't have go-cmp installed, will this change break them unless they manually install the package? Is there any way we can install it automatically when they install gotests?